readme.md

```diff
@@ -298,6 +298,10 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [upf-bpf](https://github.com/navarrothiago/upf-bpf) - An in-kernel solution based on XDP for 5G UPF.
 - [redbpf](https://github.com/foniod/redbpf) - Tooling and framework to write eBPF code in Rust efficiently.
 - [pixie](https://github.com/pixie-io/pixie) - Observability for Kubernetes using eBPF. Features include protocol tracing, application profiling, and support for distributed bpftrace deployments.
+- [Project Calico](https://projectcalico.docs.tigera.io/about/about-calico) - Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico's eBPF data plane delivers blazing fast performance with true cloud-native scalability.
```

Comment:
Do you think we could use a more objective/concrete description instead of “blazing fast performance with true cloud-native scalability”, please?

Reply:
Apologies for the marketing speak. I've toned it down and thanks for the feedback.